### PR TITLE
#3: Added installation and test instructions and privacy policy

### LINKED ISSUE #3: Installation instructions
## Acceptance Criteria

* Installation from GitHub described
* Test described

### PR DESCRIPTION
**Pull request by redcatbear**
Closes #3.

### CODE REVIEW

File: .github/workflows/ci-build.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
+    strategy:
+      fail-fast: true
+    runs-on: ubuntu-22.04
```
Comment (kaklakariada) [inline .github/workflows/ci-build.yml:13]:
```suggestion
    runs-on: ubuntu-latest
```
22.04 is the default.

File: build.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+# This simple build script determines the version number from the Manifest (to avoid
+# inconsistencies) and packs the files that make up the extension into a ZIP archive with an '.xpi'
+# suffix.
+
```
Comment (kaklakariada) [inline build.sh:5]:
```suggestion

set -o errexit
set -o nounset
set -o pipefail

```

File: doc/developer_guide/developer_guide.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+### Testing
+
+We expect GitHub to add the features into their product at some point, which will render the extension obsolete. So, we decided to skip automatic testing on this one.
+
+The test is fairly trivial anyway:
+
+1. Create a [GitHub project](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects)
+2. Create a Kanban board (the default board with its three columns is fine)
+3. Add a WiP limit to the description of the "In Progress" column by adding `(≤ 1)`
```
Comment (kaklakariada) [inline doc/developer_guide/developer_guide.md]:
```suggestion
3. Add a WiP limit to the description of the "In Progress" column by appending `(≤ 1)`
```

File: github_project_tweaks.js [hunk 4]
```diff
@@ -15,12 +15,14 @@ function getAllBoardColumns() {
     return document.evaluate(boardColumnXPath, document, null, XPathResult.ORDERED_NODE_ITERATOR_TYPE, null);
 }
 
+// Read the value of the WiP limit from the description field of the board column.
+// If there is no explicit limit, return 0.
```
Comment (kaklakariada) [inline github_project_tweaks.js:19]: Use [JSDoc style](https://jsdoc.app/about-getting-started.html) comments

**Review by kaklakariada — CHANGES_REQUESTED** [review]

File: github_project_tweaks.js (hunk 4 above)
Comment (redcatbear) [reply github_project_tweaks.js:19]: Will do in separate PR.

**Review by kaklakariada — APPROVED** [review]

**Pull request merged**